app/src/main/java/com/internship/move/feature/authentification/forgotpassword/ResetPasswordFragment.kt

```diff
@@ -1,6 +1,68 @@
 package com.internship.move.feature.authentification.forgotpassword
 
+import android.os.Bundle
+import android.view.View
+import androidx.core.widget.doOnTextChanged
 import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
 import com.internship.move.R
+import com.internship.move.databinding.FragmentResetPasswordBinding
+import com.internship.move.util.enableButton
+import com.zhuinden.fragmentviewbindingdelegatekt.viewBinding
+
+class ResetPasswordFragment : Fragment(R.layout.fragment_reset_password) {
+
+    private val binding by viewBinding(FragmentResetPasswordBinding::bind)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        initListeners()
+    }
+
+    private fun initListeners() {
+        binding.newPasswordTIL.isEndIconVisible = false
+        binding.confirmPasswordTIL.isEndIconVisible = false
+
+        binding.chevronIV.setOnClickListener {
```

Comment:
The naming `chevronIV` referst to the icon inside the button. Instead I suggest using a name that reflects the action, like `backIV` or something like that
Also, I know the design uses a chevron icon. Somethimes during development we have to push back to the project manager or designer to make some changes for the Android app that match the platform expected behavior. In this particular case, you'd want to replace the chevron with a back arrow `<-`